exporters/stats/stackdriver/README.md

```diff
@@ -67,6 +67,24 @@ public class MyMainClass {
 }
 ```
 
+#### Set Monitored Resource for exporter
```

Comment:
You need to mention that if user does not do this then will not be able to have multiple binaries sending the same metric.

Reply:
Done.